frontend/src/pages/Methodology/methodologySections/CommunitySafetyLink.tsx

```diff
@@ -0,0 +1,174 @@
+                    columns={[
+                        { header: 'Category', accessor: 'category' },
+                        { header: 'Topics', accessor: 'topic' }
+                    ]}
+                    rows={[
+                        { category: 'Community Safety', topic: communitySafetyTopicsString }
+                    ]}
+                />
+
+                <h3
+                    className='mt-12 text-title font-medium'
+                    id='#community-safety-data-sourcing'
+                >
+                    Data Sourcing
+                </h3>
+                <p>
+                    The CDC’s Web-based Injury Statistics Query and Reporting System (WISQARS)
+                    collects and provides data on gun deaths in the United States. This information
+                    is gathered from state and local injury surveillance programs and is used to
+                    understand the impact of firearm-related injuries across the country. WISQARS™
+                    allows users to explore data by intent of injury, mechanism, geographic location,
+                    and demographics, supporting public health and policy decisions.
+                </p>
+                <HetNotice className='my-12' title='A note about CDC WISQARS'>
+                    The CDC’s WISQARS limit data released at state and county levels to protect
+                    privacy. Mortality data, collected from death certificates and processed
+                    by the National Center for Health Statistics, takes approximately 13 months
+                    to become official. Provisional data and annual updates mean that reported
+                    numbers may change and should be interpreted with caution. Additionally,
+                    postcensal population estimates are updated annually, affecting fatal injury
+                    rates over time.
+                </HetNotice>
+
+                <section>
+                    <div className='py-5'>
+                        <h4 className='text-text font-normal'>WISQARS Injuries Measures</h4>
+
+                        <h5 className='my-2'>Conditions</h5>
+                        <ul className='list-inside list-disc pl-4'>
+                            <li>
+                                <HetTerm>Gun-Related deaths by Homicides</HetTerm>
+                            </li>
+                            <li>
+                                <HetTerm>Gun-Related deaths by Suicides</HetTerm>
+                            </li>
+                            <li>
+                                <HetTerm>Gun-Related deaths for Children</HetTerm>
+                            </li>
+                            <li>
+                                <HetTerm>Gun-Related deaths for Young Adults</HetTerm>
+                            </li>
+                            <li>
+                                <HetTerm>Gun-Related deaths for Black Men</HetTerm>
```

Comment:
usually we like to use sentence case, but i think title case is ok here. if that's what we are using we should capitalize `Deaths` as well